components/script/dom/servoparser/html.rs

```diff
@@ -116,18 +117,35 @@ impl Tokenizer {
     }
 }
 
-fn start_element<S: Serializer>(node: &Element, serializer: &mut S) -> io::Result<()> {
-    let name = QualName::new(None, node.namespace().clone(), node.local_name().clone());
-    let attrs = node
-        .attrs()
-        .iter()
-        .map(|attr| {
-            let qname = QualName::new(None, attr.namespace().clone(), attr.local_name().clone());
-            let value = attr.value().clone();
-            (qname, value)
-        })
-        .collect::<Vec<_>>();
-    let attr_refs = attrs.iter().map(|(qname, value)| {
+fn start_element<S: Serializer>(element: &Element, serializer: &mut S) -> io::Result<()> {
+    let name = QualName::new(
+        None,
+        element.namespace().clone(),
+        element.local_name().clone(),
+    );
+
+    let mut attributes = vec![];
+
+    // The "is" value of an element is treated as if it was an attribute and it is serialized before all
+    // other attributes. If the element already has an "is" attribute then the "is" value is ignored.
+    if !element.has_attribute(&LocalName::from("is")) {
+        if let Some(is_value) = element.get_is() {
+            let escaped_is_value = is_value.replace('&', "&amp;").replace('\u{00A0}', "&nbsp;");
```

Comment:
This is already handled by https://github.com/servo/html5ever/blob/559f96ae6ada08cc417faae71b58c800e26d2994/html5ever/src/serialize/mod.rs#L104; we should be able to pass the is_value without any modifications.